src/database/DataTypes/NameNote.js

```diff
@@ -43,6 +44,7 @@ NameNote.schema = {
     name: 'Name',
     note: { type: 'string', optional: true },
     patientEvent: 'PatientEvent',
+    storeID: { type: 'string', optional: true },
```

Comment:
@arjunSussol This variable is not optional, as it will be added by our code automatically. Set the default value to "" and remove optional please.

Reply:
@arjunSussol have you made these changes?

Reply:
@sworup done by https://github.com/openmsupply/mobile/pull/4950/commits/1c6a397c698a13c8dae25748a4519b6893621965

Reply:
@sworup I think each name_note has store_id, so it should be mandatory and have valid record.
So, we need it as `optional: false`. However, lets discuss before reaching to any point.